charts/victoria-logs-cluster/_index.md.gotmpl

```diff
@@ -20,37 +20,113 @@ tags:
 
-* PV support on underlying infrastructure.
+* **Kubernetes cluster** - A running Kubernetes cluster with sufficient resources
+* **Helm** - Helm package manager installed and configured
+
+Additional requirements depend on your configuration:
+
+* **Persistent storage** - Required if you enable persistent volumes for data retention (enabled by default)
+* **kubectl** - Needed for cluster management and troubleshooting
+
+For installation instructions, refer to the official documentation:
+* [Installing Helm](https://helm.sh/docs/intro/install/)
+* [Installing kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ## Chart Details
 
-This chart will do the following:
+### VictoriaLogs Cluster
 
-* Rollout [VictoriaLogs cluster](https://docs.victoriametrics.com/victorialogs/cluster/).
-* (optional) Rollout [vector](https://vector.dev/) to collect logs from pods.
+When you install the chart, it creates three core services and an optional log collector:
+
+- `vlinsert` receives incoming log streams from various sources and load-balances them across remote nodes.
+- `vlstorage` forms a StatefulSet that stores raw data on persistent volumes, applies the configured retention policy, and replicates the data across the number of replicas you define.
+- `vlselect` provides the query API and reads data from any storage replica, so adding more `vlselect` pods increases read throughput without impacting the stored data.
+- The log collector `vector` is disabled by default, but as soon as you set `vector.enabled: true`, a DaemonSet is deployed on every node. It starts tailing pod logs using the default parsing pipeline and automatically discovers the internal `vlinsert` addresses.
 
-Chart allows to configure logs collection from Kubernetes pods to VictoriaLogs.
-In order to do that you need to enable vector:
 ```yaml
 vector:
   enabled: true
 ```
-By default, vector will forward logs to VictoriaLogs installation deployed by this chart.
 
-Default chart setup is shown below
+The default chart setup is shown below:
+
+```mermaid
+graph LR
+    Vector["Log Collector"] --> VLI1["vlinsert-1"]
+    Vector --> VLI2["vlinsert-2"]
+    
+    subgraph "VictoriaLogs Cluster"
+        VLI1 --> VLS1["vlstorage-1"]
+        VLI1 --> VLS2["vlstorage-2"]
+        VLI2 --> VLS1
+        VLI2 --> VLS2
+        
+        VLS1 <--> VLQ1["vlselect-1"]
+        VLS1 <--> VLQ2["vlselect-2"]
+        VLS2 <--> VLQ1
+        VLS2 <--> VLQ2
+    end
+    
+    VLQ1 <--> Users["Users/Grafana/vlogscli"]
+    VLQ2 <--> Users
+```
+
+### Vector
+
+When Vector is enabled with `vector.enabled: true`, the default configuration works out of the box:
+
+* The default role is set to "Agent" (typically deployed as a DaemonSet with a data directory at `/vector-data-dir`).
+* Vector is configured with the `k8s` source, using the `kubernetes_logs` type to collect logs from all Kubernetes pods in the cluster.
```

Comment:
Could you add links to vector docs for these resources?
That would make it very easy for user to find an exact references when checking what this chart does and will help to speed up configuration troubleshooting.